coral/src/app/components/Form.tsx

```diff
@@ -408,7 +408,19 @@ function _NativeSelect<T extends FieldValues>({
             {...baseNativeProps}
             name={name}
             disabled={disabled || isSubmitting}
+            aria-readonly={props.readOnly}
             onBlur={(option) => {
+              // Prevent from setting empty string value in form when blurring out of the component
```

Comment:
For a not required select, we do not register any value in the form ^^ Essentially, this means that a form would have `undefined` as the value for the non required field, because the form object won't have a property for the optional field until one is chosen.

Reply:
💡  true!